modules/ref2vec-centroid/vectorizer/vectorizer_test.go

```diff
@@ -145,4 +145,19 @@ func TestVectorizer_Object(t *testing.T) {
 			})
 		}
 	})
+
+	t.Run("when rep prop is stored as empty string slice", func(t *testing.T) {
```

Comment:
Could you add some context here? Eg what you wrote for the fix. It is clear now, but when seeing the test in one year, we would propbably be very confused, why a reference would be an empty string :)

Reply:
sure. added